### PR TITLE
Add experimental/influxdb and experimental/usage packages.

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Review by gavincabbage — APPROVED** [review]
This is great!

Thanks for working towards documenting the measurements coming out of `usage.from`.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  host: "http://localhost:8086",
+  token: "mySupeR53cre7t0k3n",
+  headers: ["header1": "header1Value", "header2", "header2Value"],
+  query: ["ex1": "example1", "ex2": "example2"],
+  timeout: 30s,
+  body: bytes(v: "Example body")
+)
+```
+
+#### Response record schema
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:34]: Does it make sense to keep in parallel verbal phrase with the usage.from() function here, like "View response data/record schema" even though this is shorter?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+-->
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+
+## Parameters
+
+### start
+({{< req >}})
+Earliest time to include in results.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md:59]: Shall we link to/describe valid duration types or safe to assume folks know?...global

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+### path
+({{< req >}}) InfluxDB API path.
+
+_**Data type:** String_
+
+### host
+InfluxDB host URL
+_(Required when executed outside of InfluxDB)_.
+
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:58]: Does this have default (like the Cloud host does)?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+_**Data type:** String_
+
+### raw
+Return raw, high resolution usage data instead of downsampled usage data.
+Default is `false`.
+
+_**Data type:** Boolean_
+
+Depending on the resolution of queried data, `usage.from()`
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md:93]: What does this depend on (solely raw or downsampled)? If yes, we can prob drop descriptor given the example?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+| raw             |             1 hour |
+| downsampled     |            30 days |
+
+## Examples
+
+- [Query usage data for your InfluxDB Cloud organization](#query-usage-data-for-your-influxdb-cloud-organization)
+- [Query raw usage data for your InfluxDB Cloud organization](#query-raw-usage-data-for-your-influxdb-cloud-organization)
+- [Query usage data for a different InfluxDB Cloud organization](#query-usage-data-for-a-different-influxdb-cloud-organization)
+
+##### Query usage data for your InfluxDB Cloud organization
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md:107]: downsampled?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    name: influxdb.api
+    parent: InfluxDB-exp
+weight: 401
+influxdb/v2.0/tags: [functions, influxdb, api]
+---
+
+The `influxdb.api()` function submits an HTTP request to the specified InfluxDB
+API path and returns a record containing the HTTP status code, response headers,
+and response body as a byte array.
+
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:18]: This isn't really an example as much as it is showing the function and it's parameters. The examples are below. I don't love this convention, but it's what I did everywhere else. I'd like to update the convention as part of the Flux restructure.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:34]: Sure, I can do that.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+_**Data type:** String_
+
+### token
+InfluxDB authentication token
+_(Required when executed outside of InfluxDB)_.
+
+_**Data type:** String_
+
+### headers
+HTTP request headers.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:68]: These are known standards that are part of the dictionary datatype.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+_**Data type:** Dictionary_
+
+### timeout
+HTTP Request timeout.
+Default is `30s`.
+
+_**Data type:** Duration_
+
+### body
+HTTP request body.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:84]: Yes. I added a link to the `bytes()` function.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+influxdb.api(
+  method: "post",
+  path: "/api/v2/buckets",
+  host: "http://localhost:8086",
+  token: token,
+  body: bytes(v: "{
+    \"name\": \"example-bucket\",
+    \"description\": \"This is an example bucket.\",
+    \"orgID\": \"x000X0x0xx0X00x0\",
+    \"retentionRules\": []
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/influxdb/api.md:134]: Kind of. They're part of the [`/api/v2/buckets` endpoint](https://docs.influxdata.com/influxdb/v2.0/api/#operation/PostBuckets).

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/usage/from.md:59]: I think it's safe to assume users know.

**Pull request merged**